src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
```

Comment:
your card should have an inner indents (paddings) so you will only use `margin-top` for card children